packages/ui/src/lib/stores/chat.ts

```diff
@@ -6,6 +6,7 @@ export interface Message {
 	timestamp: number
 	text: string
 	address?: string
+	messageId: string
```

Comment:
Can we make this optional for now? I'm pretty sure this will break the other adapters (we care about the firebase adapter only)